docs/source/getting-started/installing.adoc

```diff
@@ -66,6 +66,7 @@ If you encounter issues related to the hypervisor, see the xref:../using/trouble
 [NOTE]
 ====
 - On Windows operating system, due to issue link:https://github.com/minishift/minishift/issues/236[#236], you need to execute the minishift binary from the drive containing your *%USERPROFILE%* directory.
+The %USERPROFILE% directory must be located in your C:\ drive, and cannot be on a network drive.
```

Comment:
@gbraad just to be extra-explicit, is it possible to run Minishift from any local drive? I don't use Windows anymore but I remember you can have multiple local drives and not just C:/.